src/dataflow/src/sink/kafka.rs

```diff
@@ -129,6 +131,47 @@ impl ProducerContext for SinkProducerContext {
     }
 }
 
+#[derive(Debug)]
+pub struct SinkConsistencyInfo {
+    topic: String,
+    schema_id: i32,
+    timestamp_counts: HashMap<Timestamp, usize>,
```

Comment:
lets just make this a map from Timestamp -> i64 to make everyone happy?
also not sure if we want to have a BTreeMap here. on the one hand -- its nice to be able to exit early, on the other hand that approach may not work when times are no longer totally ordered?